StripeFinancialConnections/StripeFinancialConnections/Source/Web/ContinueStateView.swift

```diff
@@ -41,42 +32,32 @@ class ContinueStateView: UIView {
                 "You haven't finished linking your account. Press continue to finish the process.",
                 "Title for a label explaining that the linking process hasn't finished yet."
             ),
-            contentView: {
-                let clearView = UIView()
-                clearView.backgroundColor = .clear
-                return clearView
-            }(),
-            footerView: CreateFooterView(
-                view: self
-            )
+            contentView: nil
         )
-        paneLayoutView.addTo(view: self)
     }
 
-    required init?(coder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
-    }
-
-    @objc fileprivate func didSelectContinueButton() {
-        didSelectContinue()
+    static func createFooterView(
+        didSelectContinue: @escaping () -> Void,
+        didSelectCancel: (() -> Void)? = nil
+    ) -> UIView? {
+        return PaneLayoutView.createFooterView(
+            primaryButtonConfiguration: PaneLayoutView.ButtonConfiguration(
+                title: "Continue", // TODO: when Financial Connections starts supporting localization, change this to `String.Localized.continue`,
+                action: didSelectContinue
+            ),
+            secondaryButtonConfiguration: {
+                if let didSelectCancel {
+                    return PaneLayoutView.ButtonConfiguration(
+                        title: STPLocalizedString(
+                            "Cancel",
+                            "Title of a button. It acts as a back button to go back to choosing a different bank instead of the currently selected one."
+                        ),
+                        action: didSelectCancel
+                    )
```

Comment:
non-blocking question about localization: should the "cancel" also get the same TODO treatment as continue? as-is (with a custom description) does this generate a new variant of "Cancel" to be localized?